config.py

```diff
@@ -0,0 +1,25 @@
+# Replace the values in the fields below with your own info
+
+[DATABASE]

```

Comment:
I believe this line and similar following lines will throw `NameError` since these are not defined. This is not valid Python syntax. Please replace these with comments instead.